src/ngamsPlugIns/ngamsPlugIns/eso/ngamsExtractFitsHdrDppi.py

```diff
@@ -46,7 +46,8 @@ def constructCommand(file, head=0, struct=0, skey='END', tsv=0, \
                      xmlfl='', mode=1, check=0):
     """
     """
-    cmd = '/opsw/packages/bin/printhead'
+    printhead_location = os.path.dirname(os.path.abspath(__file__))
+    cmd = printhead_location + '/printhead'
```

Comment:
This should do it

```suggestion
    cmd = sys.executable + ' ' + printhead_location + '/printhead.py'
```

Beware that `printhead.py` doesn't return non-zero on failure though, so the causes for failures (at least in py3) are not immediately obvious. For that you'll want to replace `printhead.py`'s last line from `break` to `sys.exit(1)`.